### ISSUE TITLE
Causal Transformer for Perplexity 

### ISSUE THREAD

**Issue by akshayg08** [mlfoundations/dclm#11]
Hi, 

Thanks for the great repo. I wanted to know whether the data/model used for training the 154M parameter model for calculating the perplexity is available?

**Comment by Vaishaal** [comment]
Which particular model from the paper are referring to?

**Comment by akshayg08** [comment]
Hi, I am referring to the model used for perplexity filtering mentioned in 

"For the perplexity filtering and the top-k average logits strategies, we utilize a 154M parameter causal Transformer trained on a mix of English Wikipedia, the books subset of RedPajama v1, and peS2o"

**Comment by GeorgiosSmyrnis** [comment]
Hi @akshayg08!

Perplexity filtering was one of the techniques we tried, but it is not part of the final pipeline for our dataset. As such, we have not released this model.
